packages/components/legacy_table/src/legacy_table.tsx

```diff
@@ -2,29 +2,37 @@ import { BoxProps } from "@buoysoftware/anchor-layout";
 import { CellProps, Table, TableCellConfig, TableProps } from "@buoysoftware/anchor-table";
 import { LegacyTextVariant } from "@buoysoftware/anchor-ui-legacy-text";
 import { TFunction } from "i18next";
+import get from "lodash/get";
 import snakeCase from "lodash/snakeCase";
 import { useTranslation } from "react-i18next";
 
 type LegacyTextProps = BoxProps & { variant?: LegacyTextVariant };
 type LegacyCellProps = Partial<LegacyTextProps> | undefined;
 
 type CellT = TFunction<string[], undefined, string[]>;
+type CellTP = Parameters<CellT>;
 
-type LegacyTableCellConfig<RecordData> = Omit<TableCellConfig<RecordData>, 'cellProps' | 'render'> & {
+export type LegacyTableCellConfig<RecordData> = Omit<TableCellConfig<RecordData>, 'cellProps' | 'render'> & {
   cellProps?: (rowData: RecordData) => LegacyCellProps;
   render?: (
     rowData: RecordData,
     tableT: TFunction,
-    cellT: (...params: Parameters<CellT>) => string
+    cellT: ReturnType<typeof keyPrefixedCellT>
   ) => React.ReactNode;
+  t?: TFunction<string, string | null>;
   translationNamespace?: string;
   translationKeyPrefix?: string;
 };
 
-const keyPrefixedCellT = (cellT: CellT, keyPrefix: string) => (...params: Parameters<CellT>) => {
-  const [firstParam, ...restParams] = params;
-  const prefixedParam = typeof firstParam === "string" ? [keyPrefix, firstParam].join(".") : firstParam.map(key => [keyPrefix, key].join("."));
-  return cellT(prefixedParam, ...restParams);
+const keyPrefixedCellT = (cellT: CellT, keyPrefix: string) => (key: CellTP[0], defaultValueOrOptions?: string | CellTP[1], options?: CellTP[1]) => {
```

Comment:
Realized that the translation function is overloaded, so updated this method to handle the different calls